Implmenting a web-worker for the rust merkle-tree wasm (#285)
anonklub/anonklub: workers/merkle-tree-worker/README.md

# `@anonklub/merkle-tree-worker`

> A WebWorker client-side library that is wrapping the wasm package in a web-worker to be integrated with the client-side.

Reviewer: I find it a bit redundant, what about?
```suggestion
A library that is wrapping the [`@anonklub/merkle-tree`](https://github.com/anonklub/anonklub/tree/main/merkle-tree-wasm) wasm package in a web-worker for integration into the browser client.
```
